libs/SalesforceSDKCore/SalesforceSDKCore/NativeLoginManagerTests.swift

```diff
@@ -43,29 +43,57 @@ final class NativeLoginManagerTests: XCTestCase {
     }
     
     func testUsernameValidation() async {
-        var result = await nativeLoginManager.login(username: "", password: "")
+        var result = await login(username: "", password: "")
```

Comment:
Dumb question:  How/why does this compile?

Reply:
This `login` method is a utility method of `NativeLoginManagerTests` that returned `nil` on any thrown error.  That failed the test and kept any new error handling code to a minimum.

This goes away though it we revert to using integer enums and logging on failure per an another thread on this pull request.